.travis.yml

```diff
@@ -1,20 +1,38 @@
 language: php
 
+sudo: required
+
+services:
+  - docker
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+env:
+  global:
+    - PHP_AMQP_VERSION=v1.7.0 LIBRABBITMQ_VERSION=v0.8.0
+  matrix:
+    - DEPENDENCIES=""
+    - DEPENDENCIES="--prefer-lowest --prefer-stable"

```

Comment:
Of course it's necessary. It's a good practice that many developers do.
With `--prefer-lowest` I found that the library not compatible with zend-log 2.3. You should always tests for every dependency version or, at least, for minimum and maximum versions.

Some examples:
- [ProxyManager](https://github.com/Ocramius/ProxyManager/blob/master/.travis.yml) by Ocramius
- [zend-mvc](https://github.com/zendframework/zend-mvc/blob/master/.travis.yml) Like other zf components